department-of-veterans-affairs/next-build
# VACMS-10466 nuke demo pages

## Description

Relates to #10466. Removes the demo pages as they caused a lot of extra broken link output (and increased the build time for next-build in content-build by a few minutes)

## Testing done
- ran content-build many many times with and without these pages present.
- ran `yarn export` many times with and without these pages present.

## Screenshots
n/a

## QA steps
run `yarn export` and notice it is slightly faster.
run `NODE_OPTIONS='--max-http-header-size=50000' yarn build --nosymlink --run-next-build` in the content-build repo locally and notice that the next-build portions complete much faster.
